### ISSUE TITLE
Typo in tf.keras.callbacks.EarlyStopping example

### ISSUE THREAD

**Issue by jvishnuvardhan** [tensorflow/tensorflow#44686]
## URL(s) with the issue:

Please provide a link to the documentation entry, for example:
https://www.tensorflow.org/versions/r2.0/api_docs/python/tf/MyMethod

https://www.tensorflow.org/api_docs/python/tf/keras/callbacks/EarlyStopping

## Description of issue (what needs changing):
The following is from the `EarlyStopping` page on TensorFlow website.

callback = tf.keras.callbacks.EarlyStopping(monitor='loss', patience=3)

This callback will stop the training when there is no improvement in the validation loss for three consecutive epochs.

In the above line, it should have been "loss" instead of "validation loss". 

### Clear description
Check above. Please check [this gist](https://colab.research.google.com/drive/1IL5tVIR_ulSeUQeBtDgrj0F5vZm7drYE#scrollTo=L9UTqnI3qd08) for an example.

### Correct links
https://www.tensorflow.org/api_docs/python/tf/keras/callbacks/EarlyStopping

### Submit a pull request?     
Yes

**Comment by jvishnuvardhan** [comment]
I am closing this issue as this was resolved. Thanks!
